### ISSUE TITLE
installation instructions :)

### ISSUE THREAD

**Issue by meticulo3366** [ChoudharyChanchal/game_control#6]
Please try to build a requirements file and any kind of libraries that need to be installed in the operating system. Great Project!

**Comment by ChoudharyChanchal** [comment]
> Please try to build a requirements file and any kind of libraries that need to be installed in the operating system. Great Project!

Done! 
Go through it and let me know if I can close the issue.
Thank you!

**Comment by hrm17** [comment]
Install pyobjc-framework-Quartz libray

Is this for Mac OS only ???

**Comment by vijayjc2018** [comment]
Unable to intall pyobjc-framework-Quartz in Win10; is this only for Mac OS?
Getting this ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

**Comment by ChoudharyChanchal** [comment]
> Install pyobjc-framework-Quartz libray
> 
> 
> 
> Is this for Mac OS only ??? 

Yes. For windows you have to install ctypes and there'll be a completely different directkeys.py

**Comment by ChoudharyChanchal** [comment]
Refer this link for windows OS. There is a completely different file for directkeys.py 
You'll need ctypes and it's much easier to understand as compared to MacOS

https://gist.github.com/Aniruddha-Tapas/1627257344780e5429b10bc92eb2f52a

**Comment by rongjincs** [comment]
where to download car speed fighter 2015 game on MacOS? I am running via terminal rather than spider platform. Is it fine?

**Comment by ChoudharyChanchal** [comment]
> where to download car speed fighter 2015 game on MacOS? I am running via terminal rather than spider platform. Is it fine?

I haven't tried using terminal but it shall work. 
The game is available on AppStore. You could use any other game controlled by WASD keys

**Comment by vijayjc2018** [comment]
Many thanks @ChoudharyChanchal, it worked using ctypes directkeys in Windows OS. Indeed it a good learning for me. Appreciate your effort to share the work in GitHub. Keep going!

**Comment by charanhu** [comment]
> Many thanks @ChoudharyChanchal, it worked using ctypes directkeys in Windows OS. Indeed it a good learning for me. Appreciate your effort to share the work in GitHub. Keep going!

Vijay, please share your code of directkeys.py file 
I'm also doing on windows
I getting error
please share your file

**Comment by vijayjc2018** [comment]
Hi @charanhu sharing the direct keys file for windows OS - https://github.com/vijayjc2018/GameControls/blob/master/directkeys.py

**Comment by charanhu** [comment]
Thank you for your response.
